docs/content/direct/wds.md

```diff
@@ -2,68 +2,85 @@
 
-### Using the KubeFlex CLI
+- `name` (required): A unique name for the WDS (`<wds-name>`)
+- `type` (optional, default: `k8s`):  (`<space-type>`)
 
-You can also create a WDS using the KubeFlex CLI:
+    - `k8s`: Creates a basic Kubernetes API Server with a subset of kube controllers
+    - `host`: Uses the KubeFlex hosting cluster itself. 
+    
+- `APIGroups` (optional): A comma-separated list of API Groups to include
+- `ITSName` (optional): The name of the ITS to be used by this WDS (required if multiple ITSes exist)
 
-```shell
-kflex create wds1 -p kubestellar
-```
+The type of space you choose determines the type of controllers that are included in your WDS. When creating a WDS, the recommended space types are "K8s" (the default space) and "host". If you the decide to use your hosting cluster's control plane (which is typically a KubeFlex cluster for Kubstellar) by specifying `type:host`, it presents you some benefits such as:
 
-This command creates a WDS and runs a post-create hook that deploys the KubeStellar controller manager and transport controller.
+- Saves the cost of creating an additional space.
+- Simplifies the architecture by reusing the hosting cluster
+- Makes the WDS directly accessible through the hosting cluster's API server
+
+> You can create multiple WDSes of the same space types and even a mix of different space types, where you can have the default and host WDSes in your cluster. A multiple WDS architecure can be valueable in cases where you want to manage workload runs across different sets of users or groups. Thus each user or group can use the same ITS for WEC inventory but with their different WDS
+
+### Creating a kubeconfig context for accessing the WDS
 
-## KubeFlex Hosting Cluster as WDS
+After creating your WDS, you will need to access it to submit workload objects. To do this you have to create a context for your WDS in kubeconfig context and set it context as your current cluster context:
 
-The KubeFlex hosting cluster can be configured to act as a WDS by specifying `type: host` when creating the WDS:
+- **Setup WDS context with KubeFlex CLI**: KubeFlex CLI enables you to create the context for your WDS in the kubeconfig context. Then automatically switch to it by overwiting the current kubeconfig context as your WDS context.
 
 ```shell
-helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
-  --set-json='WDSes=[{"name":"wds1", "type":"host"}]'
+# create a the WDS context and make it the current context
+kflex ctx --overwrite-existing-context <wds-name>
 ```
 
-This approach:
-- Avoids creating a separate control plane
-- Simplifies the architecture by reusing the hosting cluster
-- Makes the WDS directly accessible through the hosting cluster's API server
+- **Access WDS context with Kubernetes API**: In a case where the context for your WDS already exists in the kubeconfig context, then, you can just switch to it as your current context with the Kubernetes API. 
```

Comment:
There is no meaning to one context existing "in the kubeconfig context". I think that you mean to speak of one context existing in the (user's) kubeconfig file.